src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF_8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device_width, user_scalable=no, initial_scale=1.0, maximum_scale=1.0, minimum_scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card">
+      <div class="card__pc">
+        <div class="card__pc-img"></div>
```

Comment:
in BEM card__pc its element, you cant create element for element, check it everywhere 
```suggestion
        <div class="card__img"></div>
```